README.md

```diff
@@ -101,9 +101,11 @@ func main() {
 
 GitHub imposes a rate limit on all API clients. Unauthenticated clients are
 limited to 60 requests per hour, while authenticated clients can make up to
-5,000 requests per hour. To receive the higher rate limit when making calls
-that are not issued on behalf of a user, use the
-`UnauthenticatedRateLimitedTransport`.
+5,000 requests per hour. However, the Search API has a custom rate limit. For requests
+using authentication, you can make up to 30 requests per minute. For unauthenticated
+requests, the rate limit allows you to make up to 10 requests per minute. To receive
+the higher rate limit when making calls that are not issued on behalf of a user,
+use `UnauthenticatedRateLimitedTransport`.
```

Comment:
Some wording suggestions to consider:

- "For unauthenticated requests, you can make up to" -> "Unauthenticated clients are limited to" and "For requests using authentication, you" -> "authenticated clients" for consistency with wording above.

- "However, the Search API has a custom rate limit." Perhaps replace "however" with "Unlike all other endpoints," or just drop "However".